### PR TITLE
add new tools, from vigiechiro suite

### PR DESCRIPTION
**Pull request by Alanamosse**
New tool new PR !

3 wrappers for R scripts.
Some additional csv files are needed to perform the analyses.
Only 1 out of the 2 classifiers is strictly needed, perhaps remove the other ? (large file >10MO)

### CODE REVIEW

File: tools/vigiechiro/BilanEnrichiPFen.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<tool id="vigiechiro_BilanEnrichiPF" name="Advanced restitution: 'Point fixe' protocol" version="@VERSION@">
+    <description>from Animal Detection on Acoustic Recordings</description>
+    <macros>
+        <import>vigiechiro_macros.xml</import>
+    </macros>   
+
+    <expand macro="vigiechiro_bilanPF_bilanRP_requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
```
Comment (bgruening) [inline tools/vigiechiro/BilanEnrichiPFen.xml]: can you break this down to multiple lines to make it better readable?

Comment (Alanamosse) [reply tools/vigiechiro/BilanEnrichiPFen.xml]: Do you mean create 2 macros ? 
``vigiechiro_bilanRP_requirements``
&
``vigiechiro_bilanPF_requirements``

Comment (bgruening) [reply tools/vigiechiro/BilanEnrichiPFen.xml]: No something like this:
```
'$__tool_directory__/BilanEnrichiPF.R' 
     '$idc2' '$__tool_directory__/refPF.csv'
     '$__tool_directory__/SpeciesList.csv' 
&&
```

Comment (Alanamosse) [reply tools/vigiechiro/BilanEnrichiPFen.xml]: Oh yes sure

**Comment by Alanamosse** [comment]
@yvanlebras Are you ok with the descritpions ?

@bgruening I did the corrections. 
Also there are 2 huge files `ClassifEso*.learner`
I can get rid of one of them as it is used in a not really important test. Tell me if you prefer that way or if we can keep it.

**Comment by yvanlebras** [comment]
ok for me! THANKS @Alanamosse !

**Comment by bgruening** [comment]
@Alanamosse @yvanlebras that is your decision. 30MB for test data is a lot. It will end up in the TS and on every single Galaxy installation, including Docker images.

File: tools/vigiechiro/BilanEnrichiPFen.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            '$__tool_directory__/SpeciesList.csv' 
+        &&
+        tail -n +2 output-summary.html > tmp && mv tmp output-summary.html &&
+        tail -n +2 output-nightly.html > tmp && mv tmp output-nightly.html &&
+        tail -n +2 output-hourly.html > tmp && mv tmp output-hourly.html
+]]>
+    </command>
+    <inputs>
+        <param name="idc2" type="data" format="tabular" label="Corrected and validated observation file" help="Use output file from 'Tadarida identifications integration' : a sampling session file corrected and validated. Point fixe protocol"/>
+        <param name="output_choice" type="select" label="Select a summary display" help="Chose whether your want a nigthly, hourly or global summary">
```
Comment (bgruening) [inline tools/vigiechiro/BilanEnrichiPFen.xml]: Can this be multiple="True"? So that I get 6 outputs?

File: tools/vigiechiro/BilanEnrichiRPen.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            <filter>output_choice=='detailed'</filter>
+        </data>
+        <data name="output_detail_html" format="html" from_work_dir="output-detailRP.html" label="${tool.name} on ${on_string} Detail html">
+            <filter>output_choice=='detailed'</filter>
+        </data>
+    </outputs>
+    <tests>
+        <test>
+            <param name="idc2" value="BilanEnrichiRP_Test1.tabular"/>
+	    <param name="output_choice" value="detailed"/>
```
Comment (bgruening) [inline tools/vigiechiro/BilanEnrichiRPen.xml]: indentation

File: tools/vigiechiro/BilanEnrichiPFen.xml (hunk 2 above)
Comment (Alanamosse) [reply tools/vigiechiro/BilanEnrichiPFen.xml]: Maybe it will not be used that often, but yes I should enable multiple. Good point thank you

**Comment by Alanamosse** [comment]
I decided to remove the optional classifier (11mo).
As @bgruening suggested I enable multiple choices for outputs.
Also Thanks @yvanlebras for the English correction :)

**Comment by bgruening** [comment]
Merci!

**Pull request merged**